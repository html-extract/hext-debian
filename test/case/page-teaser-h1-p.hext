<h1/><p @text:content />
