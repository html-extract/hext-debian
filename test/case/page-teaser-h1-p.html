<h1>Heading</h1>
<p>First paragraph</p>
<p>Not included in result</p>
<p>Not included in result</p>
<h1>Heading</h1>
<p>Second paragraph</p>
<h1>Heading</h1>
<p>Third paragraph</p>
<p>Not included in result</p>
<p>Not included in result</p>
<p>Not included in result</p>
<p>Not included in result</p>
