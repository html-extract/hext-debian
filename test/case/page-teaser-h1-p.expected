{"content":"First paragraph"}
{"content":"Second paragraph"}
{"content":"Third paragraph"}
